orchestration/dap_orchestration/typing.py

```diff
@@ -0,0 +1,4 @@
+
+
+class DapSurveyType(str):
```

Comment:
Starting to add some types around all the strings we're passing around between solids.